.prettierrc

```diff
@@ -0,0 +1,11 @@
+{
+    "overrides": [
```

Comment:
And use the prettier from OZ: see https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/.prettierrc ; to follow the reference.

Reply:
Indentation fixed. Then, do you prefer to always use explicit types, as OZ's Prettier config does, or I set this option to "preserve" ? Because before editing the contracts, I saw that you were not using explicit types.

Reply:
I prefer to use the same set up than OX so with explicit types preserve.

In general, it's better to follow the reference.